src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -137,4 +137,9 @@ export class CaseHearingsListComponent implements OnInit {
   public LoadHearingRequestAndRedirect(hearingID: string, targetURL: string) {
     this.hearingStore.dispatch(new fromHearingStore.LoadHearingRequest({ hearingID, targetURL }));
   }
+
+  public LoadHearingRequestServiceHearingValuesAndRedirect(hearingID: string, targetURL: string) {
```

Comment:
@johnbenjamin-hmcts - how come the function name begins with a capital letter? I've not seen that often... if at all :) Same for the one on Line 137

Reply:
@Daniel-Lam - thanks for identifying this. I don't know why `LoadHearingRequestAndRedirect` was coded to start with a capital letter, and when I copied and created a new method `LoadHearingRequestServiceHearingValuesAndRedirect` didn't change it as well.

I have now modified both of the methods to start with a capital letter.